### DISCUSSION TITLE
Content Panel Module

### DISCUSSION THREAD

**Discussion by giscus** [hbstack/site#782] [General]
# Content Panel Module

This module display a content panel on single pages, which includes translations, comments and repo buttons.

https://hbstack.dev/modules/components/content-panel/

**Comment by FuadEfendi** [comment]
I try to set "comments: false" it doesn't seem to work...

**Reply by FuadEfendi** [reply]
My fault; it works; I was accessing cached page which was removed.

**Comment by FuadEfendi** [comment]
Docs have typo, should be `false`: "When `true` disable comments globally."

```
  docs:
    comments: false
  content_panel:
    comments: false
```

Generated HTML still has this `<div id="content-comments" class="hb-docs-doc-comments hb-module">   </div>` and it is shown as empty box on a page.

**Reply by FuadEfendi** [reply]
and the fix is to turn off comments at Page level too, 

```
cascade:
  type: docs
  comments: false
```

**Reply by razonyang** [reply]
Thank you!
